src/pages/react/platform.md

```diff
@@ -30,3 +30,24 @@ getPlatforms(); // returns ["iphone", "ios", "mobile", "mobileweb"] from an iPho
 ```
 
 Depending on what device you are on, `getPlatforms` can return multiple values. Each possible value is a hierarchy of platforms. For example, on an iPhone, it would return mobile, ios, and iphone.
+
+## Platforms
+
+Below is a table listing all the possible `Platforms` values along with corresponding descriptions.
```

Comment:
Why is `Platforms` in a code block?

Reply:
What do you mean? I don't see in a code block.

Reply:
It won't let me add a SS here but this:

> Below is a table listing all the possible `Platforms` values along with corresponding descriptions.

> `Platforms`